cacheops/local.py

```diff
@@ -0,0 +1,3 @@
+import threading
+
+thread_local = threading.local()
```

Comment:
This can then be imported by Rover to set per-request global flags – in this case, what DB the client is set to read from.